draft-ietf-taps-interface.md

```diff
@@ -1192,11 +1196,14 @@ guaranteed, any other requests are advisory.
 Type:
 : Boolean
 
-This property specifies that a message should be sent in such a way
+When true, this property specifies that a message should be sent in such a way
 that the transport protocol ensures all data is received on the other side
 without corruption. Changing the ´Reliable Data Transfer´ property on Messages
 is only possible if the Connection supports reliability.
```

Comment:
What does the 

> Connection supports reliability

 mean? The connection is using a protocol that support reliability or the application has asked for a reliable connection? If I ask for an unreliable connection and it happens to get implemented with TCP, can I ask for a reliable message?  I would say no, as this would fail most of the time. So maybe this should be "Connection is relable"?

Reply:
I agree with the idea of what you say, but not with the wording "Connection is reliable" because that sounds contradictory - we'd have a sentence saying that changing reliable data transfer per message is only possible if the Connection is reliable.  (but if it's reliable, why can you then change it?)

I think you mean that this should only be possible for Connections that were established with the Selection Property "Reliable Data Transfer (Connection)" enabled. So I propose to phrase it as: "Changing the ´Reliable Data Transfer´ property on Messages
is only possible for Connections that were established with the Selection Property 'Reliable Data Transfer (Connection)' enabled."

Please ACK or NACK - then I'll also apply this as a direct commit to the master. Sorry again for merging too fast.

Reply:
@mwelzl -Yes that is what I  I meant, thanks.

Reply:
@abrunstrom: great, thanks - I now committed both these changes to the master.